services/brig/src/Brig/TURN/API.hs

```diff
@@ -50,35 +51,61 @@ routes = do
         .&. header "Z-Connection"
 
     document "GET" "getCallsConfig" $ do
-        Doc.summary "Retrieve TURN server addresses and credentials."
+        Doc.summary "Retrieve TURN server addresses and credentials \
+                    \ for scheme `turn` and transport `udp` "
```

Comment:
double space :trollface: 

(and a trailing space, too)

Reply:
👍 :)